src/components/OverlappingCard/OverlappingCard.st.css

```diff
@@ -152,3 +146,9 @@
             BorderWidth value(borderWidth)
     );
 }
+
+.root > .mediaContainer {
+    background: linear-gradient(value(mediaBGColor), value(mediaBGColor)), linear-gradient(value(infoBGColor), value(infoBGColor));
```

Comment:
Where did the `linear-gradient`'s come from?

Reply:
@jonathanadler in order to have the same behaviour as in `Card` we can't use the same solution. because of the height different. so `linear-gradient` lets us use multiple layers of background Color

Reply:
I'm not familiar with this syntax.
Why are there 2 `linear-gradients`?
And what's the difference between this and `background-color: value(mediaBGColor);`

Reply:
you can't apply multiple layers of `background-color`  or have `background-color: value(mediaBGColor); ` and `background: linear-gradient(value(mediaBGColor), value(mediaBGColor));` at the same time.
`background` is stronger and will be shown on top of `bacgrounf-color` regardless the css order

Reply:
but `linear-gradient` after `linear-gradient` (only work with `background`) can let you apply multiple layers with order of how is on top.

Reply:
Why not use:
```
.media {
    background-color: fallback(value(mediaBgColor), value(cardBgColor));
}

.info {
    background-color: fallback(value(infoBgColor), value(cardBgColor));
}
```

Reply:
Because if the user selecting a Color with opacity the the `cardBgColor` will not be shown

Reply:
Thank you 🙏 
That was what I was missing.